content/en/agent/faq/commonly-used-log-processing-rules.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Commonly Used Log Processing Rules
```

Comment:
I'm not sure that this is a good title for this page as it doesn't really cover "common processing rules" but only different setting to deal with sensitive data. Could this be a guide  called "How to deal with sensitive data" more than a FAQ ? 
I would rather organize this page in 3 sections:
- mask sensitive data (listing all the different example  of 'mask_sequences' in sub sections)
- exclude logs including sensitive data (blacklist)
- include only safe logs (whitelist) (section to add referring to include_at_match )

Reply:
for now there are only example on how to deal with sensitive data, but soon there will be more rules documented here :) but i agree on the organization proposal